tests/test_plots.py

```diff
@@ -0,0 +1,193 @@
+import warnings
+from functools import partial, wraps
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import mudata as md
+import plotnine
+import pytest
+from matplotlib.testing.decorators import image_comparison as mpl_image_comparison
+from packaging.version import Version
+
+import prismo as pr
+
+image_comparison = partial(
+    mpl_image_comparison, extensions=["png"], tol=0.5
+)  # tolerance for differences in text rendering
+plotnine.options.base_family = "DejaVu Sans"  # bundled with Matplotlib
+
+
+@pytest.fixture(scope="module")
+def cll_data():
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", category=FutureWarning)
+        return md.read_h5mu(Path(__file__).parent / "plots" / "cll.h5mu")
+
+
+@pytest.fixture(scope="module")
+def cll_model():
+    return pr.PRISMO.load(Path(__file__).parent / "plots" / "cll_model.h5")
+
+
+@pytest.fixture(scope="module")
```

Comment:
Not `scope=session`?

Reply:
it doesn't matter at this point, since it's only used in that test file (i.e. module). If we use it somewhere else at some point, we can change it to session, but then it will probably also need to be moved out of this file.